src/Dialog/Dialog.js

```diff
@@ -1,12 +1,11 @@
-import Button from '../Button/Button';
 import classnames from 'classnames';
 import CustomPropTypes from '../utils/CustomPropTypes/CustomPropTypes';
 import FocusLock from 'react-focus-lock';
 import PropTypes from 'prop-types';
 import ReactDOM from 'react-dom';
 import React, { Component } from 'react';
-class Modal extends Component {
-    // select body element to add Modal component too
+class Dialog extends Component {
```

Comment:
Doesn't need to happen here, but this could be a functional component pretty easy too.